### ISSUE TITLE
DOMAINS router?

### ISSUE THREAD

**Issue by fboucquez** [SteveLTN/https-portal#293]
Hi @SteveLTN , thanks for your project!

Is it possible to have 2 services behind 2 domains like: 

'service1.mydomain.io -> http://service1:100/'  
'service2.mydomain.io -> http://service2:200/'

service1 and service2 are 2 containers running on the same compose. Both domains would point to the same ip. 

There are some guides related but not quite:

https://github.com/SteveLTN/https-portal#multiple-domains ()
https://github.com/SteveLTN/https-portal#serving-static-sites (2 domain but to static files)

**Comment by SteveLTN** [comment]
Yes, you can. I should update the document to be more conclusive.
Just do

DOMAINS: “a.com -> http://seevice1:1000, b.com -> http://service2:8080”

**Comment by SteveLTN** [comment]
Sure!
